### PR TITLE
ci: Fix linting getting skipped

### PR DESCRIPTION
**Pull request by edmundmiller**
Ran into with #7033 

This one has two things going on
- Linting is getting skipped
- Clean up fix for self-hosted runners

### CODE REVIEW

File: .github/workflows/gpu-tests.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
       # Prod for version bumping
     steps:
       - name: Clean Workspace # Purge the workspace in case it's running on a self-hosted runner
-        uses: ./.github/actions/self-hosted-cleanup
+        run: ./.github/scriptscleanup-selfhosted-runner.bash
```
Comment (mashehu) [inline .github/workflows/gpu-tests.yml:52]:
```suggestion
        run: ./.github/scripts/cleanup-selfhosted-runner.bash
```

File: .github/workflows/nf-test.yml [hunk 2]
```diff
@@ -63,7 +63,7 @@ jobs:
       SENTIEON_ENCRYPTION_KEY: ${{ secrets.SENTIEON_ENCRYPTION_KEY }}
     steps:
       - name: Clean Workspace # Purge the workspace in case it's running on a self-hosted runner
-        uses: ./.github/actions/self-hosted-cleanup
+        run: ./.github/scriptscleanup-selfhosted-runner.bash
```
Comment (mashehu) [inline .github/workflows/nf-test.yml:66]:
```suggestion
        run: ./.github/scripts/cleanup-selfhosted-runner.bash
```

File: .github/workflows/nf-test.yml [hunk 3]
```diff
@@ -74,4 +74,4 @@ jobs:
           shard: ${{ matrix.shard }}
           total_shards: ${{ env.TOTAL_SHARDS }}
       - name: Clean Workspace # Purge the workspace in case it's running on a self-hosted runner
-        uses: ./.github/actions/self-hosted-cleanup
+        run: ./.github/scriptscleanup-selfhosted-runner.bash
```
Comment (mashehu) [inline .github/workflows/nf-test.yml:77]:
```suggestion
        run: ./.github/scripts/cleanup-selfhosted-runner.bash
```

**Review by mashehu — APPROVED** [review]

**Comment by edmundmiller** [comment]
That's as far as I'm getting. Hopefully the runners recover over night 🫠

**Pull request merged**